benchmark/comparisonTest/main.go

```diff
@@ -51,6 +51,13 @@ func main() {
 		errMsg := fmt.Sprintf("Failed to read csv file %s with error:%v\n", configCsv, err)
 		panic(errMsg)
 	}
+
+	logFile, err := os.OpenFile(outputDir+"/benchmark_log", os.O_RDWR|os.O_CREATE, 0664)
```

Comment:
here we create the logfile, it will be recreated everytime we run benchmarks - similar to the results file